gh-actions/release-images/action.yaml

```diff
@@ -14,15 +14,12 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - shell: bash
-      env:
-        IMAGES_ARGS: --nocache ${{ inputs.image_args }}
-      run: |
-        echo "::group::Build new images"
-        make images
-        echo "::endgroup::"
```

Comment:
Yes, we can keep them — we can use the same approach as that used with buildx (export multi-arch images as tarballs).

Reply:
I think we should if it's not a hassle, since it'll be easier to understand if the image building itself failed (e.g. due to some error downloading/installing) or if the upload failed (e.g. wrong creds, service unavailable, etc)